README.md

```diff
@@ -0,0 +1,212 @@
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/default.asp">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/default.asp">CSS</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+    <li><a href="https://react.dev/learn">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[User authentication through username]**
+- **[Navigation panel with links to classes]**
+- **[Reservation form]**
+- **[My reservations page.]**
+- **[Add class and Delete class options]**
+- **[Display a list of selected classes on the main page]**
+- **[Detailed class view with a "Reserve" button]**
+- **[Responsive design for both mobile and desktop]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+ - [N/A]()
+
+## 🔗 Back End Link <a name="documentation"></a>
+- [EduPlanner Backend](https://github.com/ab-noori/EduPlanner-Backend)
+  
+  > _Our backend code is stored in a separate repository and is built using ruby on rails. The backend is responsible for processing data and handling API requests from the frontend. To access our backend code, please click on the provided link below._
+
+ 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [Ruby on Rails](https://rubyonrails.org/) installed and running. To get more information, read the [installation guide](https://guides.rubyonrails.org/).
+
+- [PostgreSQL](https://www.postgresql.org/) installed and running
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+##
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  https://github.com/ab-noori/EduPlanner-Frontend.git
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Install this project with:
+
+```sh
+    cd EduPlanner-Frontend
+    npm install
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
```

Comment:
- [ ] No frontend tests have been included:

![image](https://github.com/ab-noori/EduPlanner-Frontend/assets/57421823/d46ad85f-a12e-4b3b-b0e7-76a069c0292c)


Kindly, either include some tests or remove the section to prevent confusion. 👍